packages/lesswrong/components/sunshineDashboard/SuggestAlignmentList.jsx

```diff
@@ -0,0 +1,62 @@
+import { Components, registerComponent, withList } from 'meteor/vulcan:core';
+import React, { Component } from 'react';
+import { Posts } from 'meteor/example-forum';
+import { withStyles } from '@material-ui/core/styles';
+import withUser from '../common/withUser';
+import PropTypes from 'prop-types';
+
+const styles = theme => ({
+  root: {
+    opacity:.2,
+    '&:hover': {
+      opacity: 1,
+    }
+  },
+  icon: {
+    marginRight: 4
+  }
+})
+
+
+class SuggestAlignmentList extends Component {
+  render () {
+    const { results, classes } = this.props
+    if (results && results.length) {
+      return (
+        <div className={classes.root}>
+          <Components.SunshineListTitle>
+            <Components.OmegaIcon className={classes.icon}/> Suggestions for Alignment
+          </Components.SunshineListTitle>
+          {this.props.results.map(post =>
+            <div key={post._id} >
+              <Components.SuggestAlignmentItem post={post}/>
+            </div>
+          )}
+        </div>
+      )
+    } else {
```

Comment:
I don't think this else statement actually does anything. Javascript without returns return null if I remember correctly.

Reply:
Components need to return null or they fuck up if I recall correctly.

Reply:
Yep, you're correct. My bad.